scripts/java-api-check.sh

```diff
@@ -63,8 +65,10 @@ if [[ -z $RELEASE_TAG ]]; then
     AAR_PATH="s3://mapbox-api-downloads-production/v2/mobile-maps-android-base/releases/android/"${LAST_VERSION}"/maven/base-"${LAST_VERSION}".aar"
   fi
   echo "aar path is: $AAR_PATH, checking file"
-  #todo check if file exists on s3
-#  aws s3 ls "$AAR_PATH" >/dev/null 2>&1
+  if [[ -z $(aws s3 ls "$AAR_PATH") ]]; then
+    echo "$AAR_PATH doesn't exist on s3, skipping the check"
```

Comment:
Would be also nice to skip the check if the acual major release is triggered?
Seems not implemented now and will fail the CI. Though it's definitely very occasional and won't happen many times, but still.
Up to you to decide

Reply:
We have the script for this scenario https://github.com/mapbox/mapbox-maps-android/blob/main/scripts/semver-check.sh

Reply:
Hmm, this semver-check checks if version was updated correctly, right? 
I mean slightly different thing - we dont want to fail on API breaking change PR if it's PR with major update, e.g. v9-v10

Reply:
This job isn't mandatory, it fails to notify there is api breaking changes but will not block other jobs or merge a PR.